clients/src/main/resources/common/message/ConsumerGroupHeartbeatResponse.json

```diff
@@ -39,7 +39,7 @@
     { "name": "ErrorMessage", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
       "about": "The top-level error message, or null if there was no error." },
     { "name": "MemberId", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
-      "about": "The member id generated by the coordinator. Only provided when the member joins with MemberEpoch == 0." },
+      "about": "The member id is generated by the consumer and provided by the consumer for all requests." },
```

Comment:
hmm... I feel that we don't need to specify the behavior in version `0` since it's an unstable version. 
If users are using the 4.0 release, they should have the latest behavior.

> However, in version 1+, it is basically the value received from the consumer?

I think yes.